articles/getting-started/install-configure-bonsai.md

```diff
@@ -0,0 +1,84 @@
+---
+uid: BonsaiInstallationAndConfiguration
+title: Bonsai Installation and Configuration
+---
+
+## Install Bonsai
+
+To download Bonsai, select between the portable download and the installer download [here](https://bonsai-rx.org/docs/articles/installation.html).
+
+*   The **Portable** download installs a sandboxed version of Bonsai. Portable environments enable users to switch between different environments to prevent package conflicts or confusion between similar packages.
+
+    *   To install from the **Portable** download, extract the downloaded file. You can start the portable Bonsai by running the `Bonsai.exe` that is inside the extracted folder.
```

Comment:
Does it actually install the .exe directly for the portable download, or do they need to run an install script? I'm not at my computer yet, but I can check once I get there